fjord/base/tests/test_google_utils.py

```diff
@@ -0,0 +1,101 @@
+from functools import wraps
+import sys
+
+from mock import MagicMock, patch
+from nose.tools import eq_
+import requests_mock
```

Comment:
Yeah... I waffle between that and putting them in order alphabetically. At one point, we were throwing together a linter for it, but then I decided I just don't care a whole lot.

But I'll fix it and write it down in the docs so I remember in the future.